server/modules/turf/turf.controller.ts

```
@@ -146,99 +176,133 @@
         },
       };
     } catch {
-      return { error: new ErrorResponse("Invalid operatingHours format", 400) };
+      return { error: new ErrorResponse('Invalid operatingHours format', 400) };
     }
   }
 
-  private parseTeamSize(value: any, next: NextFunction) {
+  private parseTeamSize(value: any) {
     if (!value) return { data: {} };
     const parsed = parseInt(value);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)